src/appkit.rs

```diff
@@ -1025,36 +1024,37 @@ impl NSWindow for id {
 
     unsafe fn initWithContentRect_styleMask_backing_defer_(self,
                                                            rect: NSRect,
-                                                           style: NSUInteger,
+                                                           style: NSWindowMask,
                                                            backing: NSBackingStoreType,
                                                            defer: BOOL) -> id {
         msg_send![self, initWithContentRect:rect
-                                  styleMask:style
+                                  styleMask:style.bits
                                     backing:backing as NSUInteger
                                       defer:defer]
     }
 
     unsafe fn initWithContentRect_styleMask_backing_defer_screen_(self,
                                                                   rect: NSRect,
-                                                                  style: NSUInteger,
+                                                                  style: NSWindowMask,
                                                                   backing: NSBackingStoreType,
                                                                   defer: BOOL,
                                                                   screen: id) -> id {
         msg_send![self, initWithContentRect:rect
-                                  styleMask:style
+                                  styleMask:style.bits
                                     backing:backing as NSUInteger
                                       defer:defer
                                      screen:screen]
     }
 
     // Configuring Windows
 
-    unsafe fn styleMask(self) -> NSUInteger {
-        msg_send![self, styleMask]
+    unsafe fn styleMask(self) -> NSWindowMask {
+        let styleMask = msg_send![self, styleMask];
+        styleMask
```

Comment:
I'm surprised that this works as written. Doesn't this need to call NSWindowMask::from_bits or from_bits_truncate?